src/Network/Wai/Log/Options.hs

```diff
@@ -106,17 +123,17 @@ defaultLogResponse uuid req resp responseBody time =
                          ]
     ]
 
--- | Helper to consistently log the UUID in your application by adding
--- @request_uuid@ field to log's 'localData'
-logRequestUUID :: MonadLog m => UUID -> m a -> m a
-logRequestUUID = localData . requestUUID
+-- | Helper to consistently log the request id in your application by adding
```

Comment:
Just voicing my opinion, but I believe that `logRequestID` and `requestID` nor their `UUID` counterparts shouldn't exist.
Not only it doesn't have configurable label, but when you change `defaultLogResponse` or `defaultLogRequest`, it can get out of sync when you keep using it. Such function is just slightly out of scope of this library.

Just the fact that we're extracting an `UUID`/`id` from the request and forcing it's presence in the signatures is what caused need for this PR in the first place. If we simply didn't care about any ids in this library, then we could just modify `logRequest` and `logResponse` and that would be it.